extras/wezterm/README.md

```
@@ -0,0 +1,20 @@
+# Aquatermium.wez ⌨️
+##### Aquavium color scheme on WezTerm
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)